packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -972,6 +972,15 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 			}
 		}
 
+		function onSetAttrib(event: any) {
```

Comment:
Just one thing - for new code please only use arrow function expressions (`const onSetAttrib = (...)`). This is because `function` implicitly creates a `this` variable inside the function, and that has made refactoring more difficult in the past.

Reply:
Fixed.